### ISSUE TITLE
Onfido framework conflicts with Segment Analytics

### ISSUE THREAD

**Issue by fmessina** [onfido/onfido-ios-sdk#121]
### What was the expected behaviour?

I know this has been reported and fixed in the past, but I am still experiencing this issue.
It looks like the Segment Analytics sources are compiled into the Onfido framework, which leads to compatibility issues at runtime.

### Integration configuration:
```
  pod 'Onfido', '16.0.0'
  ## Analytics
  pod 'Analytics', '3.7.0'
  pod 'Segment-Adjust', '2.0.0'
  pod 'Segment-Mixpanel', '1.4.0'
```

### Steps to reproduce:

The following warning appears in console for each Segment Analytics class

```
objc[18340]: Class PodsDummy_Analytics is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e23d8) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bee60). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGAES256Crypto is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2400) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bee88). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGAliasPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2450) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014beed8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGAnalytics is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e24a0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bef28). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGAnalyticsConfiguration is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e24f0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bef78). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGContext is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2540) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014befc8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGFileStorage is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2590) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf018). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGGroupPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e25e0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf068). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGHTTPClient is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2630) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf0b8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGIdentifyPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2680) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf108). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGIntegrationsManager is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e26d0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf158). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGBlockMiddleware is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2720) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf1a8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGMiddlewareRunner is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2770) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf1f8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e27c0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf248). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGApplicationLifecyclePayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2838) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf2c0). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGRemoteNotificationPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2888) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf310). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGContinueUserActivityPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e28d8) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf360). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGOpenURLPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2928) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf3b0). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGReachability is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2950) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf3d8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGScreenPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e29a0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf428). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGSegmentIntegration is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e29f0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf478). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGSegmentIntegrationFactory is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2a40) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf4c8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGStoreKitTracker is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2a90) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf518). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGTrackPayload is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2ae0) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf568). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGUserDefaultsStorage is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2b30) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf5b8). One of the two will be used. Which one is undefined.
objc[18340]: Class SEGUtils is implemented in both /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Onfido.framework/Onfido (0x1029e2ba8) and /private/var/containers/Bundle/Application/B327653B-EE3E-4658-82F8-1E5F5A9EAD7F/WealthApp.app/Frameworks/Analytics.framework/Analytics (0x1014bf630). One of the two will be used. Which one is undefined.
```

**Comment by onfido-ios-team** [comment]
Hello @fmessina , 
We have fixed this issue and will deliver it in the next release.
I will inform you from here once we released.

Thanks,
Kerem

**Comment by keremgund** [comment]
Hello @fmessina ,
We have fixed this issue and will deliver it in the next release.
I will inform you from here once we released.

Thanks,
Kerem

**Comment by keremgund** [comment]
Hello @fmessina ,
Sorry for the late reply, this issue has been fixed with 16.1.0 release. 
https://github.com/onfido/onfido-ios-sdk/releases/tag/16.1.0

Please let me know if you need further support.

Thanks,
Kerem

**Comment by fmessina** [comment]
Hi, thanks for the update. 
I can confirm the the issue has been fixed in the `16.1.0` release.
